types/ocpp.yaml

```diff
@@ -2,6 +2,62 @@ description: >-
   OCPP types (OCPP1.6 and OCPP2.0.1). The types are based more on the type definitions of OCPP201,
   as these offer more flexibility and are easier to transfer to OCPP1.6 than vice versa.
 types:
+  ChargingSchedulePeriod:
+    description: >-
+      Element providing information on a charging schedule period.
+    type: object
+    required:
+      - startPeriod
+      - limit
+      - stackLevel
+    properties:
+      startPeriod:
+        type: integer
+      limit:
+        type: number
+      numberPhases:
+        type: integer
+      stackLevel:
+        type: integer
+  ChargingSchedule:
+    description: >-
+      Element providing information on an OCPP charging schedule.
+    type: object
+    required:
+      - connector
+      - chargingRateUnit
+      - chargingSchedulePeriod
+    properties:
+      connector:
+        type: integer
+        minimum: 0
```

Comment:
I think it might be a good idea to already include the evse in this type definition in preparation for usage of this type with OCPP 2.0.1

Reply:
added evseid as optional for OCPP 2.0.1